docs/advanced/api-v2/rest.md

```diff
@@ -0,0 +1,4019 @@
+---
+title: REST API V2
+---
+
+FORMAT: 1A
+HOST: https://api.counterparty.io
+
+# Counterparty Core API
+
+The Counterparty Core API is the recommended (and only supported) way to query the state of a Counterparty node. 
+
+API routes are divided into 11 groups:
+- [`/blocks`](#group-blocks)
+- [`/transactions`](#group-transactions)
+- [`/addresses`](#group-addresses)
+- [`/assets`](#group-assets)
+- [`/orders`](#group-orders)
+- [`/bets`](#group-bets)
+- [`/dispensers`](#group-dispensers)
+- [`/burns`](#group-burns)
+- [`/events`](#group-events)
+- [`/mempool`](#group-mempool)
+- [`/backend`](#group-backend)
+
+Notes:
+
+- When the server is not ready, that is to say when all the blocks are not yet parsed, all routes return a 503 error except `/` and those in the `/blocks`, `/transactions` and `/backend` groups which always return a result.
+
+- All API responses contain the following 3 headers:
+
+    * `X-COUNTERPARTY-HEIGHT` contains the last block parsed by Counterparty
+    * `X-BACKEND-HEIGHT` contains the last block known to Bitcoin Core
```

Comment:
`X-BITCOIN-HEIGHT`

... and everywhere else :)

Reply:
done!